storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operations/execute/PreparedExecuteSQL.java

```diff
@@ -113,7 +118,22 @@ public Object executeAsBlocking() {
     @CheckResult
     @Override
     public Observable<Object> asRxObservable() {
-        return RxJavaUtils.createObservable(storIOSQLite, this);
+        throwExceptionIfRxJavaIsNotAvailable("asRxObservable()");
+
+        final Set<String> observesTables = rawQuery.observesTables();
+        final Set<?> observesTags = rawQuery.observesTags();
+
+        final Observable<Object> observable;
+        if (!observesTables.isEmpty() || !observesTags.isEmpty()) {
+            observable = ChangesFilter.applyForTablesAndTags(storIOSQLite.observeChanges(), observesTables, observesTags)
```

Comment:
Yes, I definitely remember discussing that, and I'm pretty sure we have docs about it :)